lib/cn/cn_tree.c

```diff
@@ -1668,6 +1670,8 @@ cn_comp_update_split(
 
             assert(route_node_keycmp(w->cw_split.key, w->cw_split.klen, right->tn_route_node) < 0);
 
+            right->tn_split_ns = get_time_ns() + 60 * NSEC_PER_SEC;
+            right->tn_split_ns += (right->tn_split_ns % 1048576) * 32;
```

Comment:
This prevents a given node from splitting multiple times in a short time frame, which happens early in the load.  This allows the node to accumulate more keys for better subquent key distribution on a subsequent split, and also helps to reduce scatter and tiny kvsets early in the load.